.fixtures.yml

```diff
@@ -1,10 +1,12 @@
 fixtures:
   repositories:
-    'facts': 'git://github.com/puppetlabs/puppetlabs-facts.git'
-    'provision': "git://github.com/puppetlabs/provision.git"
-    'puppet_agent': 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
-    'stdlib': "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+    facts: 'https://github.com/puppetlabs/puppetlabs-facts.git'
+    provision: "https://github.com/puppetlabs/provision.git"
+    puppet_agent: 
+      repo: 'https://github.com/puppetlabs/puppetlabs-puppet_agent.git'
+      ref: v4.13.0
+    stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
   forge_modules:
-    'mount_core': "puppetlabs/mount_core"
+    mount_core: "puppetlabs/mount_core"
```

Comment:
I think it makes sense to switch to the git repo here as well

Reply:
Updated. 👍